component/loki/source/gcplog/internal/gcplogtarget/push_translation.go

```diff
@@ -4,22 +4,24 @@ package gcplogtarget
 // configure and run the targets that can read log entries from cloud resource
 // logs like bucket logs, load balancer logs, and Kubernetes cluster logs
 // from GCP.
+// /!\ But
+// - one line is edited because of dependencies version diff (-> ctrl+f 'edited')
+// - import 'github.com/grafana/agent/component/common/loki' is added, and api.* is replace by loki.*
 
 import (
 	"encoding/base64"
 	"fmt"
 	"strings"
 
 	"github.com/grafana/agent/component/common/loki"
-	"github.com/grafana/loki/pkg/util"
 	"github.com/prometheus/common/model"
 	"github.com/prometheus/prometheus/model/labels"
 	"github.com/prometheus/prometheus/model/relabel"
-)
 
-// ReservedLabelTenantID reserved to override the tenant ID while processing
-// pipeline stages
-const ReservedLabelTenantID = "__tenant_id__"
```

Comment:
Let's add this back in instead of relying on importing the `lokiClient` package just for a string definition.